src/features/classification/security-controls/__tests__/SecurityControls.test.js

```diff
@@ -59,6 +59,14 @@ describe('features/classification/security-controls/SecurityControls', () => {
         expect(wrapper).toMatchSnapshot();
     });
 
+    test('should render label for security controls when shouldRenderLabel prop is set', () => {
+        wrapper.setProps({ controlsFormat: SHORT });
```

Comment:
I think we might not actually want the label for some of formats, but I guess that would be up to the parent component?

Reply:
Yes, that's how it's implemented.